lib/constructs/callback-interface.js

```diff
@@ -178,14 +197,41 @@ class CallbackInterface {
   }
 
   generateInstall() {
+    if (this.constants.size > 0) {
+      this.str += `
+        const exposed = new Set([
+      `;
+
+      for (const globalName of this.exposed) {
+        this.str += `"${globalName}",\n`;
+      }
+
+      this.str += `
+        ]);
+      `;
```

Comment:
What about just `const exposed = new Set(${JSON.stringify([...this.exposed])});`